core-bundle/tests/Fixtures/Functional/Routing/suffix.yml

```diff
@@ -0,0 +1,2 @@
+sql:
```

Comment:
We don't ususally do that in functional tests because it introduces dependencies between fixtures. If someone adjusts the original fixture file loaded before, it might break this test as well. It would be better to have one completeley independent fixture for every test case. Just leaving this here as general statement. You may do with it what you want :)